src/App.jsx

```diff
@@ -14,33 +16,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_NAME = 'name';
+const SORT_FIELD_LENGTH = 'length';
+const ENTER_GOODS = goodsFromServer;
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function prepareSortedGoods(goods, sortField, reverse) {
```

Comment:
The function name 'prepareSortedGoods' should start with a verb according to the checklist item: 'Method's name should start with a verb'. Consider renaming it to something like 'getSortedGoods'.